packages/devtools_app/lib/src/screens/profiler/cpu_profile_controller.dart

```diff
@@ -458,13 +454,15 @@ class CpuProfilerController
       _dataNotifier.value = await processDataForTag(tag).then(
         (e) => e.getActive(viewType.value),
       );
-    } catch (e) {
+    } catch (e, stackTrace) {
       // In the event of an error, reset the data to the original CPU profile.
       final filteredOriginalData = cpuProfileStore.lookupProfile(
         label: _wrapWithFilterSuffix(userTagNone),
       )!;
       _dataNotifier.value = filteredOriginalData.getActive(viewType.value);
-      throw Exception('Error loading data with tag "$tag": ${e.toString()}');
+      Error.throwWithStackTrace(
+          Exception('Error loading data with tag "$tag": ${e.toString()}'),
+          stackTrace);
```

Comment:
nit: tc

Reply:
Done. Interestingly this is a case the DCM trailing comma lint will flag. As a follow up I may try enabling it.